packages/integration-tests/src/playbook/contracts/L1.sol

```diff
@@ -0,0 +1,12 @@
+//SPDX-License-Identifier: Unlicense
```

Comment:
It should be called just `TestERC20` or something like that, and why do you use two separate contracts for L1 and L2, it should be the same one, just deploy it with different args if you want.